pkg/network/usm/debugger/README.md

```diff
@@ -0,0 +1,30 @@
+# USM debugger
+
+A minimal, self-contained build of USM for faster iterations of debugging eBPF
+code on remote machines.
+
+Prepare the build for the target architectures you need (the `foo` KMT stack
```

Comment:
I'd add a comment here that you need this for the eBPF object files, specially with cross-arch builds.

Reply:
Done.